omod/src/main/webapp/admin/concepts/conceptDrugForm.jsp

```diff
@@ -10,9 +10,8 @@
 <script type="text/javascript">
 	dojo.require("dojo.widget.openmrs.ConceptSearch");
 	dojo.require("dojo.widget.openmrs.OpenmrsPopup");
-
 	<request:existsParameter name="autoJump">
-		var autoJump = <request:parameter name="autoJump"/>;
+		var autoJump = ${openmrs:getSafeJsBoolean(param.autoJump)};
```

Comment:
@dkayiwa the following url will trigger an alert message when using the <c:out> tag. I tested on the latest version of firefox:
`http://localhost:8080/openmrs/admin/concepts/conceptDrug.form?autoJump=false;alert(1);`

However, during testing I realized there was a bug in my filtering function (oops!) so I'm going to close this PR and re-open with revised changes.